ios/config.json

```diff
@@ -18,5 +18,10 @@
     "HLLocalization.bundle",
     "HLResources.bundle",
     "KonotorModels.bundle"
-  ]
+  ],
+  "plist": {
+    "NSMicrophoneUsageDescription": "This game uses the microphone, so you can send us voice feedback",
```

Comment:
can we have better wording here? Like

"We need access to microphone to send us a voice feedback"